### ISSUE TITLE
Valgrind memcheck errors triggered by GeoIP

### ISSUE THREAD

**Issue by yurivict** [maxmind/geoip-api-c#60]
````
==75111== 1 errors in context 1 of 1:
==75111== Conditional jump or move depends on uninitialised value(s)
==75111==    at 0x4E446C6: ??? (in /usr/local/lib/libGeoIP.so.1.6.5)
==75111==    by 0x4E44A5E: _GeoIP_seek_record_gl (in /usr/local/lib/libGeoIP.so.1.6.5)
==75111==    by 0x4E4550D: GeoIP_country_code_by_name_gl (in /usr/local/lib/libGeoIP.so.1.6.5)
==75111==    by 0x4E47960: GeoIP_country_code_by_name (in /usr/local/lib/libGeoIP.so.1.6.5)
````

**Comment by oschwald** [comment]
Do you have the calling code that can reproduce this, including the initialization of the database struct?

**Comment by yurivict** [comment]
````
#include "GeoIP.h"
#include <stdio.h>

int main() {
  GeoIP *gi = GeoIP_open("/usr/local/share/GeoIP/GeoIP.dat", GEOIP_STANDARD | GEOIP_CHECK_CACHE);
  const char* cc = GeoIP_country_code_by_name(gi, "8.8.8.8");
  printf("cc=%s\n", cc);
}
````

**Comment by oschwald** [comment]
This was fixed in #60.
